### PR TITLE
Add commands support for 'gg' and 'G'

### PR DESCRIPTION
**Pull request by liushuping**
Add navigation commands support:
 - 'gg': go to fist line, on first non-blank character
 - 'G': go to last line, on first non-blank character

### CODE REVIEW

**Comment by mmcdole** [comment]
:+1:

**Comment by jpoon** [comment]
Would you mind adding tests? See [this](https://github.com/VSCodeVim/Vim/blob/master/test/cursor.test.ts) as a reference.

Regressions are starting to become an issue :(

**Comment by liushuping** [comment]
Tests have been added and functions are renamed to 'firstLineNonBlanChar' and 'lastLineNonBlanChar' to differentiate them from document begin and end functions. Please review.

File: src/cursor.ts [hunk 1]
```diff
@@ -224,4 +235,8 @@ export default class Cursor {
 
 		return null;
 	}
+
+	private static posOfFirstNonBlankChar(line: number): number {
```
Comment (jpoon) [inline src/cursor.ts]: In the latest for `master`, the is a similar implementation for this. Would you be able to reuse the `getNextWordPosition` or `getPreviousWordPosition` functions?

**Comment by jpoon** [comment]
One last thing, can you squash your commits: `git rebase -i HEAD~4`. Thanks.

**Comment by liushuping** [comment]
Update: commits are squashed to a single one

**Comment by jpoon** [comment]
:+1:

**Pull request merged**